akka-http-core/src/main/resources/reference.conf

```diff
@@ -52,6 +52,12 @@ akka.http {
     # The time period within which the TCP binding process must be completed.
     bind-timeout = 1s
 
+    # Default value for the http port on which the server will be running.
```

Comment:
"Default port to bind HTTP server to when no port was explicit given."

Reply:
should it be "explicitly"? I'm not a native speaker but explicit sounds weird here.

Reply:
should it be "explicitly" here? I'm not a native speaker but explicit sounds weird here ;)

Reply:
I'd say it's explicitly in this case.

Reply:
Yeah, I pushed a new change yesterday with "explicitly".